lib/Sema/SemaDecl.cpp

```diff
@@ -13537,7 +13537,6 @@ Decl *Sema::ActOnTag(Scope *S, unsigned TagSpec, TagUseKind TUK,
 
   if (Attr)
     ProcessDeclAttributeList(S, New, Attr);
-  ProcessAPINotes(New);
```

Comment:
Whoops! That was supposed to be a separate patch (with a good explanation). Thanks for catching this!